src/service/resources-impl.js

```diff
@@ -787,6 +799,34 @@ export class ResourcesImpl {
     }
   }
 
+  /** @override */
+  getEagerElementRatio() {
+    if (this.totalLayoutCount_ === 0) {
+      return 0;
+    }
+
+    let eagerCount = 0;
+    for (let i = 0; i < this.resources_.length; i++) {
```

Comment:
Can't this be computed in `taskComplete_` with slow elements?

Reply:
i dont think so. the value changes over time as the viewer scrolls and some of the elements are no longer considered eager.